ui/pages/routes/routes.component.js

```diff
@@ -122,6 +96,66 @@ import {
   showOnboardingHeader,
 } from './utils';
 
+// Begin Lazy Routes
+const OnboardingFlow = React.lazy(() =>
+  import('../onboarding-flow/onboarding-flow'),
+);
+const Lock = React.lazy(() => import('../lock'));
+const UnlockPage = React.lazy(() => import('../unlock-page'));
+const RestoreVaultPage = React.lazy(() => import('../keychains/restore-vault'));
+const RevealSeedConfirmation = React.lazy(() =>
+  import('../keychains/reveal-seed'),
+);
+const Settings = React.lazy(() => import('../settings'));
+const NotificationsSettings = React.lazy(() =>
+  import('../notifications-settings'),
+);
+const NotificationDetails = React.lazy(() => import('../notification-details'));
+const Notifications = React.lazy(() => import('../notifications'));
+const SnapList = React.lazy(() => import('../snaps/snaps-list'));
+const SnapView = React.lazy(() => import('../snaps/snap-view'));
+
+// eslint-disable-next-line -- Un-lazy this one for now
```

Comment:
Perhaps more readable to have an alphabetical list of lazy routes, and then an alphabetical list of standard routes defined with comments?

Reply:
Yeah this is in the middle of the list to keep the same order as the `<Switch>` component.  Eventually I hope to figure it out and make it lazy again.